streamaggr: ignore first intervals (#6137)
VictoriaMetrics/VictoriaMetrics: docs/stream-aggregation.md

- To set `ignore_old_samples: true` option at the particular [aggregation config](#stream-aggregation-config).
  This enables ignoring old samples for that particular aggregation config.

## Ignore intervals after start

> When there's a samples queue in a component (e.g: vmagent) before a one, which performs aggregation (vmagent or vminsert) sometimes there's a need to ignore initial intervals not to have spikes in an output. If some initial intervals should be ignored, then the following options can be used:

Reviewer: ```suggestion
Stream aggregation may yield inaccurate results if it processes incomplete data. This issue can arise when data is sourced from clients that maintain a queue of unsent data, such as Prometheus or vmagent. If the queue isn't fully cleared within the aggregation interval, only a portion of the time series may be included in that period, leading to distorted calculations. To mitigate this, consider the following options:
```

> - To pass `-remoteWrite.streamAggr.ignoreFirstIntervals=<intervalsCount>` command-line flag to [vmagent](https://docs.victoriametrics.com/vmagent/)

Reviewer: ```suggestion
- Set `-remoteWrite.streamAggr.ignoreFirstIntervals=<intervalsCount>` command-line flag to [vmagent](https://docs.victoriametrics.com/vmagent/)
```

>   or `-streamAggr.ignoreOldIntervals=<intervalsCount>` command-line flag to [single-node VictoriaMetrics](https://docs.victoriametrics.com/).

Reviewer: ```suggestion
  or `-streamAggr.ignoreFirstIntervals=<intervalsCount>` command-line flag to [single-node VictoriaMetrics](https://docs.victoriametrics.com/) to skip first `<intervalsCount>` [aggregation intervals](#stream-aggregation-config)
  from persisting to the storage.  It is expected that all incomplete or queued data will be processed during 
  specified `<intervalsCount>` and  all subsequent aggregation intervals will produce correct data.
```

>   This enables ignoring first intervals for all the [aggregation configs](#stream-aggregation-config).

Reviewer: ```suggestion
```

> - To set `ignore_old_intervals: <intervalsCount>` option at the particular [aggregation config](#stream-aggregation-config).

Reviewer: ```suggestion
- To set `ignore_first_intervals: <intervalsCount>` option at the particular [aggregation config](#stream-aggregation-config).
```
